src/engine/renderer/tr_shade.cpp

```diff
@@ -1240,6 +1240,8 @@ static void Render_lightMapping( int stage )
 
 	gl_lightMappingShader->SetHeightMapInNormalMap( hasHeightMapInNormalMap );
 
+	gl_lightMappingShader->SetNormalMapFromHeightMap( hasHeightMap && !hasNormalMap );
```

Comment:
Should this be done in more places? I see this attribute being added to a lot of shaders, but it is only switched on in this one place.

Reply:
Yep, it's currently only done on lightMapping shader.
I'll enable it in other places too.

Reply:
done